site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,202 @@
+
+<!--lint enable maximum-line-length-->
+
+## Integrations
+
+CVAT is a global tool, trusted and utilized by teams worldwide.
+Below is a list of key companies that contribute significantly to our
+product support or are an integral part of our ecosystem.
+
+> **Note:** If you're using CVAT, we'd love to
+> hear from you at [contact@cvat.ai](mailto:contact+github@cvat.ai).
+
+- [**Human Protocol**](https://hmt.ai): This platform incorporates CVAT
+  for augmenting its annotation services within the Human Protocol framework.
+- [**FiftyOne**](https://fiftyone.ai): As an open-source tool for dataset
+  management and model analysis in computer vision,
+  FiftyOne is [closely integrated](https://voxel51.com/docs/fiftyone/integrations/cvat.html)
+  with CVAT, enhancing annotation capabilities and label refinement.
+- [**Toloka**](https://toloka.ai): Toloka leverages CVAT for its crowdsourced data labeling
+  and annotation services, enriching its
+  diverse task handling capabilities. For mre information, see [**Integration with Toloka**](/docs/integration/toloka/)
+
+## License Information
+
+This code is distributed under the [**MIT License**](https://opensource.org/licenses/MIT),
+a permissive free software license.
+
+Additionally, this software incorporates LGPL-licensed components from
+the [**FFmpeg**](https://www.ffmpeg.org) project.
+For more details, visit [**FFMPEG Legal**](https://www.ffmpeg.org/legal.html).
+Users must independently verify whether their use of FFmpeg necessitates
+additional licenses.
+CVAT.ai Corporation does not provide these licenses
+and is not liable for any related licensing fees incurred through the use of FFmpeg.
+
+For terms of use and confidential information handling,
+see [**Terms of Use**](https://www.cvat.ai/terms-of-use).
+
+## Get in touch
+
+> **Note:** For **commercial support
+> inquiries**, please email us directly at [**contact@cvat.ai**](mailto:contact+github@cvat.ai).
+
+- [**Gitter community**](https://gitter.im/opencv-cvat/public): For questions about using CVAT.
+  This platform is actively monitored by our core team and the broader community,
+  ensuring quick responses to questions. You can also find answers to frequently asked questions here.
+
+- [**Discord channel**](https://discord.gg/S6sRHhuQ7K). For broader
+  discussions and questions, head over to our. This is a space for all things related to CVAT.
+- [**LinkedIn**](https://www.linkedin.com/company/cvat-ai/): For company updates and employment opportunities.
+
+- [**YouTube channel**](https://www.youtube.com/@cvat-ai): For tutorials and screencasts about CVAT tools.
+
+- [**GitHub issues**](https://github.com/cvat-ai/cvat/issues): To report bugs.
```

Comment:
We also have Customer support channel for CVAT.ai cloud paid users:https://youtrack.cvat.ai/form/447d9c98-ab4b-466e-bf9d-004f01b22f73

Probably we want to specify it as well. Again, need to somehow separate it from the open-source version. Please think how to do that.